pkg/driver/camera/camera_linux.go

```diff
@@ -179,6 +179,11 @@ func (c *camera) VideoRecord(p prop.Media) (video.Reader, error) {
 		return nil, err
 	}
 
+	err = c.cam.SetFramerate(float32(p.FrameRate))
```

Comment:
How it behaves when user doesn't explicitly specify frame rate?

Reply:
Good point. I will check that.

Reply:
I added some tests. When the framerate is not specified the constraint seems to be ignored.

Reply:
`p.FrameRate` seems to be 0 when user doesn't set frame rate.
I think it's better not to call SetFramerate in this case.